app/components/icon_list/icon_list.html.erb

```diff
@@ -1,5 +1,5 @@
 <ul <%= attrs %>>
-  <% I18n.t('components.lists.'.concat(list)).each do |elem| %>
```

Comment:
Maybe there was a specific reason for this implementation that I’m missing here. If that’s the case, feel free to drop this commit.

In general, I think I’d prefer explicitly passing the contents to the list via the component’s properties instead of just the I18n key. Technically, the icon list might be used for anything, that the contents come from the localization files in this particular case is irrelevant to the component.

Reply:
I think that was just a legacy from the previous `steps` component, adapted a bit to fit the new requirements. But I like this version much better.